### PR TITLE
add scaffold page for KCNA2024

### PR DESCRIPTION
**Pull request by mfahlandt**
Missing parts for the go live:

Registration Form 
Notes Folder

DO not merge before June 6th

### CODE REVIEW

File: content/en/events/2024/kcsna/faq.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+Kubernetes Orgs:
+
+<ul>
+<li><a href="https://github.com/kubernetes" rel="noopener noreferrer" target="_blank">kubernetes</a></li>
+<li><a href="https://github.com/kubernetes-client" rel="noopener noreferrer" target="_blank">kubernetes-client</a></li>
+<li><a href="https://github.com/kubernetes-csi rel="noopener noreferrer" target="_blank">kubernetes-csi</a></li>
+<li><a href="https://github.com/kubernetes-sigs" rel="noopener noreferrer" target="_blank">kubernetes-sigs</a></li>
+</ul>
+
```
Comment (mrbobbytables) [inline content/en/events/2024/kcsna/faq.md:67]: We should mention the etcd-io org here too.

File: content/en/events/2024/kcsna/faq.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+### I have an idea for a session
+
+Most of the Summit will be unconference-style, so you will have an opportunity
+to propose a session closer to the Summit itself. If you have an idea for a talk,
+workshop or panel please post your idea to the unconference issue or the CFP
+form once they become available.  
+
+### My SIG wants to meet at the Summit
+We will have some space for SIG/WG meetings during the summit, please submit
+your request via the CFP form once it becomes available.
```
Comment (mrbobbytables) [inline content/en/events/2024/kcsna/faq.md:87]:
```suggestion
### My SIG wants to meet at the Summit

We will have some space for SIG/WG meetings during the summit, please submit
your request via the CFP form once it becomes available.
```

Just a small nit, but figured can fix when updating the org member section^^;;

File: content/en/events/2024/kcsna/registration.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+
+
+**NOTE:** The summit adheres to KubeCon’s [Health & Safety Guidelines]
+
+If you acknowledge the above, please go ahead and click below to register:
+
+<h3>
+<a href="https://cvent.me/Ak8z00?locale=en-US&tm=5BFaB2eHY0xZyx_vVZdfLzrXFWPJJsz___EvVtztajE&rp=D599C1DE-92F0-4EC4-8DA2-82B18574E7EF" rel="noopener noreferrer" target="_blank">Register Here</a>
+</h3>
```
Comment (mrbobbytables) [inline content/en/events/2024/kcsna/registration.md:23]: Should replace with 'registration will open soon' for now. That links to an old dead reg page.

More so a note for ourselves when we DO add the reg links. We need to make it super explicit that non-org members MUST have a sig lead reach out to us before they register. Or whatever process we want to do. Just not have them + a lead into the convo >_>

File: content/en/events/2024/kcsna/social.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+weight: 5
+---
+Join us for the Contributor Social
+from 6pm to 9pm for an evening celebration after the Contributor Summit on November 11th!
+
+There will be food, live music, activities, and more!
+
+*The contributor social is _not_ a public event.  You must be [registered] in advance to attend the Contributor Summit in order to attend the Social, and must be a Kubernetes contributor.  Community members who are not registered and show up at the Social will be turned away.*
+
+[registered]: /events/2023/kcsna/registration/
```
Comment (mrbobbytables) [inline content/en/events/2024/kcsna/social.md:13]:
```suggestion
[registered]: /events/2024/kcsna/registration/
```

File: content/en/events/2024/kcsna/registration.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Registration"
+type: docs
+weight: 2
+---
+
+Registration is open to all members of one of the [Kubernetes orgs], qualified Etcd contributors, or sponsored attendees.
+Registration closes: **Thursday, November 7th, 2023; 16:00 US Eastern Time**
```
Comment (mrbobbytables) [inline content/en/events/2024/kcsna/registration.md:8]:
```suggestion
Registration closes: **Thursday, November 7th, 2024; 16:00 US Eastern Time**
```

File: content/en/events/2024/kcsna/registration.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+**NOTE:** The summit adheres to KubeCon’s [Health & Safety Guidelines]
+
+If you acknowledge the above, please go ahead and click below to register:
+
+<h3>
+<a href="https://cvent.me/Ak8z00?locale=en-US&tm=5BFaB2eHY0xZyx_vVZdfLzrXFWPJJsz___EvVtztajE&rp=D599C1DE-92F0-4EC4-8DA2-82B18574E7EF" rel="noopener noreferrer" target="_blank">Register Here</a>
+</h3>
+
+[Kubernetes orgs]: /events/2023/kcsna/faq/#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person
```
Comment (mrbobbytables) [inline content/en/events/2024/kcsna/registration.md:25]:
```suggestion
[Kubernetes orgs]: /events/2024/kcsna/faq/#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person
```

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks for getting this together 🙏 
Made a few suggestions, mostly where the year is still set to 2023. I made the same mistake all over the place last time so its always one of the first things I check lol

File: content/en/events/2024/kcsna/faq.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+- [New Contributor Workshop 2019 - Barcelona](https://www.youtube.com/watch?v=BQ7y2TFOzF4&list=PL69nYSiGNLP2WTJ6P8sQenhf0RY-JqF5L)
+- [New Contributor Workshop 2019 - San Diego](https://www.youtube.com/watch?v=uUJrGwAom-E&list=PL69nYSiGNLP0OWp38tPBc-jSlMmwWr6Ci&index=15)
+- [Kubernetes Contributor Guide](/docs/guide/)
+
+### Can I bring a guest to the Social?
+
+Contributors will be allowed to bring a single family member
+or partner to the social (a "plus one"). 
+[See the Social page to request a guest pass](https://www.kubernetes.dev/events/2024/kcsna/social/).
+
```
Comment (jberkus) [inline content/en/events/2024/kcsna/faq.md:110]:
```suggestion

### What about Utah HB 257?

Recent laws passed by the state of Utah may cause problems for some members of our community.  Please see the [legal information] on the Venue page.

[legal information]: link-to-Laws-section-of-venue-page
```

Comment (stmcginnis) [reply content/en/events/2024/kcsna/faq.md:110]: I believe we should link to https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/attend/diversity-inclusion/#utah-bathroom-bill-statement, but I think making a statement like "may cause problems for some members of our community" should probably be kept out of it. That statement somewhat conflicts with the text on the referenced page:

> To clarify the legislation, this law does not extend to privately owned venues, including those hosting our conference and related activities

Better to make sure folks have easy access to the vetted information than to make any statements here.

**Comment by stmcginnis** [comment]
Looks good!

/lgtm

**Pull request merged**